docs/fastdds/xml_configuration/common.rst

```diff
@@ -308,15 +301,15 @@ Liveliness
 ReliabilityQosPolicy
 """"""""""""""""""""
 
-+---------------------------+----------------------------------+----------------------+--------------------------------+
-| Name                      | Description                      | Values               | Default                        |
-+===========================+==================================+======================+================================+
-| ``<kind>``                | See                              | |BEST_EFFORT|        | DataReaders:                   |
-|                           | :ref:`reliabilityqospolicykind`. +----------------------+ |BEST_EFFORT| |br|             |
-|                           |                                  | |RELIABLE|           | Datawriters: |RELIABLE|        |
-+---------------------------+----------------------------------+----------------------+--------------------------------+
-| ``<max_blocking_time>``   | See :ref:`reliabilityqospolicy`. | :ref:`DurationType`  | 100 ms                         |
-+---------------------------+----------------------------------+----------------------+--------------------------------+
++------------------------+----------------------------------+----------------------+-----------------------------------+
+| Name                   | Description                      | Values               | Default                           |
++========================+==================================+======================+===================================+
+| ``<kind>``             | See                              | |BEST_EFFORT-xml-api|| DataReaders: |BEST_EFFORT-xml-api||
+|                        | :ref:`reliabilityqospolicykind`. +----------------------+                                   |
+|                        |                                  | |RELIABLE-xml-api|   | DataWriters: |RELIABLE-xml-api|   |
```

Comment:
Remove the line between datareaders and datawriters in the last columns. 
To break lines in tables, use the alias `|br|`.